docs/src/data/test/props-table.test.ts

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
+
+test('Props Table', () => {
+  const propsTableData = fs.readFileSync('./src/data/props-table.json', 'utf8');
```

Comment:
I'm still not sure why we are checking in this file if we have a snapshot test of it. We could ignore this file from git, it gets built during the `build` script and then gets tested against the snapshot for changes. I'm wary of checking this file into git if its going to be overridden when the docs get built in PRs and deploys especially since it is so large.

Reply:
oh, good point. I'll gitignore it.

Reply:
Done!